### PR TITLE
bump to polkadot-stable2407-3

### PR DESCRIPTION
**Pull request by clangenb**
Companion of https://github.com/ajuna-network/ajuna-parachain/pull/70

We are probably not going to deploy this on Bajun anymore, but as long as Bajun exists, I thought it makes sense to update the code base.

### CODE REVIEW

File: rust-toolchain.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel    = "1.78.0"
+channel    = "1.81.0"
```
Comment (DidacSF) [inline rust-toolchain.toml:2]: Is there any reason we upgraded to `1.81.0`?

Comment (clangenb) [reply rust-toolchain.toml:2]: it is the same as parity uses, and it is stable. Do you have any concerns?

Comment (DidacSF) [reply rust-toolchain.toml:2]: Nope, just wondering! Thanks!

File: .github/workflows/runtimes-matrix.json [hunk 2]
```diff
@@ -4,12 +4,5 @@
...
     "path": "runtime/bajun-runtime",
     "uri": "wss://rpc-parachain.bajun.network:443"
-  },
-  {
-    "name": "bajun-paseo",
-    "package": "bajun-runtime",
-    "path": "runtime/bajun-runtime",
-    "uri": "wss://rpc-paseo.bajun.network:443",
-    "is_relay": false
   }
```
Comment (clangenb) [inline .github/workflows/runtimes-matrix.json:7]: Paseo does no longer exist.

**Review by DidacSF — APPROVED** [review]

**Pull request merged**